arch/arm64/boot/dts/qcom/sm7325-nothing-spacewar.dts

```diff
@@ -930,7 +930,14 @@
 };
 
 &cci1_i2c0 {
-	/* actuator (For Rear-aux UW sensor) @ 0xc */
+	camu_dw9800w: actuator@c {
+		compatible = "dongwoon,dw9800w",
+			     "dongwoon,dw9800";
+		reg = <0x0c>;
+
+		vin-supply = <&vreg_cam_vio_1p8>;
+		vdd-supply = <&vreg_camu_vaf_1p8>;
```

Comment:
My DW9800K only has a VDD pin (and VSS - whatever that does - is connected to ground), what is VIN for you?

Reply:
Product briefs for related ICs:
* DW9800S https://hmisemi.com/app/uploads/2023/11/DW9800S_Product-Brief_R1.0_0622.pdf
* DW9800K https://hmisemi.com/app/uploads/2023/11/DW9800K_Product-Brief_R1.1_0722.pdf
* DW9800V http://dwanatech.com/wp-content/themes/generatepress_child/images/popup_download/DW9800V.pdf

Reply:
Downstream driver, as well as upstream driver of similar IC has both of these regulators

Reply:
https://github.com/Moto-MT6879-Dev/kernel_motorola_mt6879/blob/14/drivers/misc/mediatek/lens/vcm/v4l2/dw9800w/dw9800w.c#L46

Reply:
Upstream:
* dw9714: vcc-supply
* dw9719: vdd-supply
* dw9768: vin-supply & vdd-supply
* dw9807: no regulator support

But since the dw9800 in all its variants I have seen only has vdd, this should mean only vdd-supply should be added to driver/dt-bindings